fanficfare/adapters/adapter_asianfanficscom.py

```diff
@@ -113,6 +113,23 @@ def doAdultCheck(self, url, soup):
         else:
             return False
 
+    def doSubCheck(self, url, soup):
```

Comment:
This method isn't a check--it's a change to both the site and returns a value.  Instead of `doSubCheck`, perhaps `doStorySubscribe`?  'Check' implies it doesn't change anything and returns boolean to me.

Reply:
Honestly the entire method was based on doAdultCheck - I haven't paid much attention to the logic at this point. I'll implement the changes accordingly.